spydrnet_physical/util/ConnectPoint.py

```diff
@@ -155,12 +161,18 @@ def direction(self, reverse=False):
     @staticmethod
     def _rotate_point(point, angle, sizex=None, sizey=None):
         x, y = point
-        if angle in (90, ):
+        if angle in (90, -270):
             return(sizex-y+1, x)
-        elif angle in (180, ):
+        elif angle in (180, -180):
             return(sizex-x+1, sizey-y+1)
-        elif angle in (270, ):
+        elif angle in (270, -90):
             return(y, sizey-x+1)
```

Comment:
We have added the negative angles in this function but it is still not working. Can you please check this?